src/lib/utilities/Popup/popup.ts

```diff
@@ -82,11 +82,11 @@ export function popup(node: HTMLElement, args: PopupSettings) {
 		focusableElems = Array.from(elemPopup?.querySelectorAll(elemWhitelist));
 		// reset the focus index
 		activeFocusIdx = -1;
-		// Automatically focus the element if openWithFocus is true (for example if
-		// the menu was opened with Enter instead of with a click
-		activeFocusIdx = 0;
 		// if the popup was triggered via focus, we don't want to move that focus
+        // we also only want to set activeFocusIdx to 0 if the popup wasn't triggered via `focus` or `focus-click`
 		if (args.event !== 'focus' && args.event !== 'focus-click') {
+            // Automatically focus the element if the event is not `focus` or `focus-click`
+            activeFocusIdx = 0;
```

Comment:
If you fix the intendations, I'll even give you a ✅  @JustBarnt  ;)

Reply:
I could perhaps do that ;)

Reply:
@Sarenor Was it because I use spaces instead of tabs? I just ran prettier nothing looked different so I assume prettier is set to use tabs 🤣